src/meltano/core/bundle/settings.yml

```diff
@@ -191,16 +191,19 @@ settings:
   value: ["https://sp.meltano.com"]
 
 # Feature Flags
+# Control whether lock files are enabled or not.
 - name: ff.lock_files
   kind: boolean
   value: false
-
-# Feature Flags
+# Control whether to use Uvicorn rather than Gunicorn as the API server.
 - name: ff.enable_uvicorn
   kind: boolean
   value: False
-
-# Feature Flags
+# Control whether the API includes scheduled jobs in listings.
```

Comment:
Possibly as a follow-on but would be useful if we could put the comment text as `description: <>`.

I don't know if this is supported though, and it's not worth blocking on. (Fwiw, I checked the yaml file and no 'description' entries exist.)

Reply:
> I don't know if this is supported though,

It supported in the sense that I can add it and it doesn't break anything. But I don't think we have anything that would automatically make use of it or expose it, at least that I'm aware of.